packages/reactor-browser/src/operational/hooks/useLiveQuery.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  type LiveQuery,
+  type LiveQueryResults,
+} from "@electric-sql/pglite/live";
+import { type Kysely } from "kysely";
+import { useEffect, useState } from "react";
+import { usePGliteDB } from "../../pglite/hooks/usePGlite.js";
+import { useOperationalStore } from "./useOperationalStore.js";
+
+export type QueryCallbackReturnType = {
+  sql: string;
+  parameters?: readonly unknown[];
+};
+
+export function useLiveQuery<Schema, T = unknown, TParams = undefined>(
```

Comment:
Let's name it `useOperationalQuery` to make it clear.

Reply:
fixed